### PR TITLE
9.0 mrp workorder link

### PR DESCRIPTION
**Pull request by thibaultrey**
Add workorder link module that link between to workcenter.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This is already covered in the module _mrp_operations_extension_ of the version 8, so this will duplicate the functionality.

File: mrp_workcenter_workorder_link/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    'version': '0.5',
+    'author': 'Akretion',
+    'summary': "Switch easily between Work Centers and Work Orders",
+    'maintener': 'Akretion',
+    'category': 'Manufacturing',
+    'depends': [
+        'mrp_operations',
+    ],
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (sebastienbeau) [inline mrp_workcenter_workorder_link/__openerp__.py:34]: move to readme file

File: mrp_workcenter_workorder_link/static/description/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (sebastienbeau) [inline mrp_workcenter_workorder_link/static/description/index.html:1]: remove as we use the readme file

**Comment by sebastienbeau** [comment]
@pedrobaeza I will try the mrp_operations_extension module. Thx

**Comment by LoisRForgeFlow** [comment]
@thibaultrey @florian-dacosta branch has been deleted. Good to close?

**Comment by LoisRForgeFlow** [comment]
Closing as this is stale. Please reopen if you think still is relevant.

**Pull request closed without merging**